src/analyzer/analyzerqueue.cpp

```diff
@@ -458,10 +450,11 @@ void AnalyzerQueue::slotAnalyseTrack(TrackPointer tio) {
 
 // This is called from the GUI and from the AnalyzerQueue thread
 void AnalyzerQueue::queueAnalyseTrack(TrackPointer tio) {
```

Comment:
I think we should remove legacy uses of "tio" and variants throughout the code. I'd be surprised if they didn't confuse new developers unfamiliar with the old name of Track objects.